### DISCUSSION TITLE
Open Dropdown when related Label is clicked - not working

### DISCUSSION THREAD

**Discussion by bhaidar** [primefaces/community#400] [PrimeVue] [answered]
Hi,

I have a Dropdown with a Label. I am connecting them through teh "for" and "inputId".

I would like to open the Dropdown list when the label is clicked.

Check attached video. Thanks

https://github.com/primefaces/community/assets/1163421/e96e4c95-4dcf-4feb-9d94-fc8f35adfbbb

Bill

**Comment by FlipWarthog** [comment, marked as answer]
An issue was created for this which contains a manual workaround. If the linked PR lands, the workaround will no longer be needed. https://github.com/primefaces/primevue/issues/4567
